### PR TITLE
Support for MS8607 PHT (Pressure Humidity Temperature) sensor

### PR DESCRIPTION
**Pull request by e28eta**
## Description:

Documentation for new component MS8607

**Related issue (if applicable):** fixes esphome/feature-requests#932

**Pull request in [esphome](https://github.com/esphome/esphome) with YAML changes (if applicable):** esphome/esphome#3307

## Checklist:

  - [x] Branch: `next` is for changes and new documentation that will go public with the next ESPHome release. Fixes, changes and adjustments for the current release should be created against `current`.
  - [x] Link added in `/index.rst` when creating new documents for new components or cookbook.

I ran the image compression suggestion from https://esphome.io/guides/contributing.html and the file sizes I see are:

- `images/ms8607.jpg` 19k, with pixel dimensions 300x216 
- `components/sensor/images/ms8607-full.jpg` 150k, with pixel dimensions 774x556

### CODE REVIEW

**Comment by e28eta** [comment]
I’m still hopeful the functional PR and docs PR land

**Comment by e28eta** [comment]
I’m still hopeful the functional PR and docs PR land

**Comment by e28eta** [comment]
The force push 5 minutes ago was a no-conflict rebase onto the head of next, e189d6d85c5096a4ca890a9f88aef056bc16f9b2

File: components/sensor/ms8607.rst [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+- **humidity** (**Required**): The information for the humidity sensor.
+
+  - **name** (**Required**, string): The name for the humidity sensor.
+  - **id** (*Optional*, :ref:`config-id`): Set the ID of this sensor for use in lambdas.
+  - **address** (*Optional*, int): Manually specify the I²C address of
+    the humidity sensor. Defaults to ``0x40``.
+  - **i2c_id** (*Optional*, :ref:`config-id`): Manually specify the ID of the :ref:`I²C Component <i2c>` if your
+    configuration uses multiple I²C buses. This should match the ``i2c_id`` documented below.
+  - All other options from :ref:`Sensor <config-sensor>`.
```
Comment (jesserockz) [inline components/sensor/ms8607.rst]:
```suggestion
- **temperature** (**Required**): The information for the temperature sensor.
  All options from :ref:`Sensor <config-sensor>`.

- **pressure** (**Required**): The information for the pressure sensor.
  All options from :ref:`Sensor <config-sensor>`.

- **humidity** (**Required**): The information for the humidity sensor.
  - **address** (*Optional*, int): Manually specify the I²C address of
    the humidity sensor. Defaults to ``0x40``.
  - **i2c_id** (*Optional*, :ref:`config-id`): Manually specify the ID of the :ref:`I²C Component <i2c>` if your
    configuration uses multiple I²C buses. This should match the ``i2c_id`` documented below.
  - All other options from :ref:`Sensor <config-sensor>`.
```

**Review by jesserockz — CHANGES_REQUESTED** [review]

**Review by jesserockz — DISMISSED** [review]

**Review by jesserockz — APPROVED** [review]

**Pull request merged**